graph-db/serverless.yml

```diff
@@ -82,3 +76,10 @@ resources:
         PolicyName: cmrParamStorePolicy-${opt:stage}
         Roles:
           - Ref: IamRoleGraphLambdaExecution
+custom:
+  variables:
+    # Default values for environment variables
+    CMR_ROOT: ${env:CMR_ROOT, 'https://cmr.sit.earthdata.nasa.gov'}
```

Comment:
I would suggest this is just blank if no default is set to make sure its being done explicitly as part of a deployment

Reply:
I am going to change it to be deduced from `stage`.

Reply:
Actually, we do want the ability to set a `CMR_ROOT` that is different from what `stage` maps to. And I don't want to have to put in `CMR_ROOT` every time for the default deployment `sit`. So I am going to keep it the way it was.